src/main/java/seedu/duke/commands/ShowRateCommand.java

```diff
@@ -0,0 +1,19 @@
+package seedu.duke.commands;
+
+import seedu.duke.Currency;
+
+public class ShowRateCommand extends Command {
+    private Currency from;
+    private Currency to;
+    public ShowRateCommand(String from, String to) {
+        super(false);
+        this.from = Currency.valueOf(from);
+        this.to = Currency.valueOf(to);
```

Comment:
These 2 lines throw an Exception when the currency does not exist. Do remember to catch the error prior to passing it in should the currency be invalid.

Reply:
I will leave implementation of tests to the person who is fully implementing the command in #16

Reply:
Agreed